src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -28,9 +28,32 @@ def list(self):
 
     def get(self, fs_id):
         fs_id = self.fs_id_to_int(fs_id)
-
         return self.fs_status(fs_id)
 
+    @RESTController.Resource('GET')
+    def cd_details(self, fs_id):
```

Comment:
As per discussion this endpoint will be renamed to `dashboard_details`.

```suggestion
    def dashboard_details(self, fs_id):
```

Reply:
also please look at `dashboard.controllers.pool.Pool#_info`

Reply:
I will create a separate PR for that change